### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sgratzl**

### CODE REVIEW

File: src/FormBuilder.ts [hunk 1]
```diff
@@ -241,19 +241,14 @@ abstract class AFormElement extends EventHandler implements IFormElement {
...
       return;
     }
 
-    for(const key in attributes) {
-      // skip loop if the property is from prototype
-      if(!attributes.hasOwnProperty(key)) {
-        continue;
-      }
-
+    Object.keys(attributes).forEach((key) => {
```
Comment (thinkh) [inline src/FormBuilder.ts]: Good idea! 👍

File: src/FormBuilder.ts [hunk 2]
```diff
@@ -406,18 +410,20 @@ class FormSelect extends AFormElement implements IFormSelectElement {
...
-        const values = dependElements.map((d) => d.value);
-        optionsData = options.optionsFnc(values);
+      const values = <IFormSelectOption[]>dependElements.map((d) => d.value);
+      optionsData = options.optionsFnc(values);
 
-        depElem.on('change', (evt, value) => {
-          const values = dependElements.map((d) => d.value);
-          this.updateOptionElements(options.optionsFnc(values));
-          $select.property('selectedIndex', options.selectedIndex || 0);
+      const onDependentChange = ()=> {
```
Comment (thinkh) [inline src/FormBuilder.ts]: Missing space between `()=>`

**Comment by sgratzl** [comment]
@chrilehner your review is missing

File: src/FormBuilder.ts [hunk 3]
```diff
@@ -82,17 +82,17 @@ export class FormBuilder {
    * @param id
    * @returns {IFormElement}
    */
-  getElementById(id:string):IFormElement {
```
Comment (chrilehner) [inline src/FormBuilder.ts]: is there a reason for the removed return type?

Comment (sgratzl) [inline src/FormBuilder.ts]: the return type is automatically inferred, so there is no need to declare it

**Pull request merged**